src/libsystemd/sd-event/test-event.c

```diff
@@ -17,6 +17,8 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#include <sys/wait.h>
```

Comment:
`WEXITED` - glibc defines it in `stdlib.h` if `_GNU_SOURCE` or `_XOPEN_SOURCE` is defined.

X/OPEN specification says that some macros from `sys/wait.h` should be also defined in `stdlib.h`. However, `WEXITED` is not on that list. glibc seems to define it by accident.